<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>626</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>626</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>626</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Расчет количества норм удобрений</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(141, 196, 145);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="potassium_cb">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="culture_cb">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>530</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantity_tb">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Book Antiqua&quot;;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="load_img_btn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>119</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>Обработать изображение</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLabel" name="phosphor_lbl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>Содержание в почве фосфорных соединений</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="phosphor_cb">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="square_lbl">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>139</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>Площадь, га</string>
    </property>
   </widget>
   <widget class="QLabel" name="potassium_lbl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>Содержание в почве калийных соединений</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="square_tb">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>129</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Book Antiqua&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="MainWindow_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>19</y>
      <width>1061</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="quantity_lbl_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Book Antiqua&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Количество колосьев на 1 кв. м, шт.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="number_lbl_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>180</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Book Antiqua&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Число зёрен в колосе, шт.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="number_tb">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>190</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Book Antiqua&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);

</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="mass_lbl_2">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>180</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Book Antiqua&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Масса 1000 зёрен, г</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="mass_tb">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>190</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Book Antiqua&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="title_lbl">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>671</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Book Antiqua</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Расчет количества норм удобрений</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="culture_lbl">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>Сельскохозяйственная культура</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>530</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <zorder>MainWindow_2</zorder>
   <zorder>square_tb</zorder>
   <zorder>clear_btn</zorder>
   <zorder>potassium_lbl</zorder>
   <zorder>culture_lbl</zorder>
   <zorder>load_img_btn</zorder>
   <zorder>phosphor_cb</zorder>
   <zorder>phosphor_lbl</zorder>
   <zorder>calc_btn</zorder>
   <zorder>culture_cb</zorder>
   <zorder>quantity_tb</zorder>
   <zorder>square_lbl</zorder>
   <zorder>potassium_cb</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
